lib/matplotlib/mlab.py

```diff
@@ -2867,14 +2867,19 @@ def get_func(name, item, func):
         'print':  'print_',
         }
 
-    def get_converters(reader):
+    def get_converters(reader, comments):
 
         converters = None
-        for i, row in enumerate(reader):
+        i = 0
+        for row in reader:
+            if (len(row) and comments is not None and
+                    row[0].startswith(comments)):
```

Comment:
Actually, the same test is used in the main loop of `csv2rec`. So unless we change it everywhere (which might break things), I would leave it as it is.